app/code/Magento/InventoryConfigurableProductAdminUi/Block/Adminhtml/Product/Steps/Bulk.php

```diff
@@ -98,13 +98,11 @@ private function generateJsonConfig(UiComponentInterface $data, string $provider
         }
 
         if (!empty($children)) {
-            $config[$name]['children'] = $config[$name]['children'] ?? [];
+            $config[$name]['children'] = $config[$name]['children'] ?? [[]];
```

Comment:
@larsroettig: probably `$config[$name]['children'] = [$config[$name]['children'] ?? []];` will be more correct here (the original element might be a flat array, so semantics will change).